bin/local-env/start.sh

```diff
@@ -15,6 +15,11 @@ cd "$(dirname "$0")/../.."
 # Check whether Composer installed
 . "$(dirname "$0")/install-composer.sh"
 
+# Add GB trunk to /tmp/gutenberg
+rm -rf /tmp/gutenberg /tmp/gutenberg.zip
+wget https://gutenbergtimes.com/wp-content/uploads/2020/11/gutenberg.zip -O /tmp/gutenberg.zip
```

Comment:
Where'd you get this URL from? Is this an official nightly URL?

Reply:
It's not official. I just used it for testing purposes.